README.md

```diff
@@ -0,0 +1,21 @@
+# mlflow-elasticsearchstore
+
+Mlflow plugin to use Elaticsearch backend in the Tracking Store of Mlflow. To use this plugin you need a running instance of Elasticsearch 6.X.
+
+## Development
+
+In a python environment :
```

Comment:
"in the same python env where you installed mlflow" ?

Reply:
Not needed, mlflow is reinstalled with the requirements.txt if needed